cemu/ui/main.py

```diff
@@ -634,7 +669,7 @@ def refreshWindowTitle(self) -> None:
     def showShortcutPopup(self):
         """Display a popup with all shortcuts currently defined"""
         msgbox = QMessageBox(self)
-        msgbox.setWindowTitle("CEMU Shortcuts from: {:s}".format(CONFIG_FILEPATH))
+        msgbox.setWindowTitle("CEMU Shortcuts from: {}".format(CONFIG_FILEPATH))
```

Comment:
can even go f-strings
```suggestion
        msgbox.setWindowTitle(f"CEMU Shortcuts from: {CONFIG_FILEPATH}")
```